src/index.js

```diff
@@ -0,0 +1,131 @@
+// ------------- Wave 2 ----------------------
+const state = {
+  temp: 60,
+  city: 'Seattle',
+  sky: '☁️',
+};
+// 1. increase temp
+const addDegree = (event) => {
+  state.temp += 1;
+  console.log('state temp', state.temp);
+  const tempContainer = document.querySelector('#degrees');
+  tempContainer.textContent = state.temp;
+  updateColorsAndEmojis();
+};
+// 2. decrease temp
+const subtractDegree = (event) => {
+  state.temp -= 1;
+  console.log('state temp', state.temp);
+  const tempContainer = document.querySelector('#degrees');
+  tempContainer.textContent = state.temp;
+  updateColorsAndEmojis();
+};
```

Comment:
These work great! Can you consider how you could make your code more DRY by combining them into a single function?